pom.xml

```diff
@@ -51,6 +51,7 @@
     <!-- Docker component versions -->
     <docker.alpine.version>3.16.2</docker.alpine.version>
     <postgres.version>12.12-alpine</postgres.version>
+    <solr.version>8</solr.version>
```

Comment:
We're using an older version because we determined there were API changes that would break the front-end if we upgraded?

Reply:
I figured we'd use 8 for consistency, since we just migrated several apps to 8

Reply:
Up to you. Solr describes 8 as "Previous major version _may sometimes_ [emphasis mine] receive critical bugfix releases"

Reply:
The Postgres version specified in the POM is older too, btw, which I assumed we were also doing for consistency with other apps.

Reply:
@cachemeoutside do you see any other reason (deployment-related or otherwise) why we shouldn't use Solr 9?

Reply:
Is this project going to use a centralized postgres or spin up its own? For some reason I thought we were spinning up our own Solr but using the centralized postgres(?)

Reply:
I have no objection for solr 9. I don't think we plan on running a centralized solr going forward due to prior issues with updating

Reply:
Solr 9 appears to be the latest stable version. I'd say switch to it if you can.

Reply:
Switched